kfdef/kfctl_aws.yaml

```diff
@@ -7,417 +7,98 @@ spec:
   - kustomizeConfig:
       repoRef:
         name: manifests
-        path: application/application-crds
-    name: application-crds
-  - kustomizeConfig:
-      overlays:
-      - application
-      repoRef:
-        name: manifests
-        path: application/application
+        path: application/v3
     name: application
   - kustomizeConfig:
-      parameters:
-      - name: namespace
-        value: istio-system
-      repoRef:
-        name: manifests
-        path: istio-1-3-1/istio-crds-1-3-1
-    name: istio-crds
-  - kustomizeConfig:
-      parameters:
-      - name: namespace
-        value: istio-system
       repoRef:
         name: manifests
-        path: istio-1-3-1/istio-install-1-3-1
-    name: istio-install
+        path: stacks/aws/application/istio-1-3-1-stack
+    name: istio-stack
   - kustomizeConfig:
-      parameters:
-      - name: namespace
-        value: istio-system
       repoRef:
         name: manifests
-        path: istio-1-3-1/cluster-local-gateway-1-3-1
+        path: stacks/aws/application/cluster-local-gateway-1-3-1
     name: cluster-local-gateway
   - kustomizeConfig:
-      parameters:
-      - name: clusterRbacConfig
-        value: 'ON'
       repoRef:
         name: manifests
-        path: istio/istio
+        path: istio/istio/base
```

Comment:
em.. we probably want to create a stack for all istio manifests later

Reply:
yeah makes sense, we can also do so for cert-manager later